cli/joy.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+'use strict';
```

Comment:
Let's discuss our use of single vs double quotes for consistency.

Reply:
Cool. IMHO single for JS, double for JSON. Or use backticks. Easily enforceable via eslint